config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get 'customers/:id', to: 'customers#show', as: 'customer'
```

Comment:
I'm curious why you added the `as:` option here. It's not necessary to make this work and typically using the restful route is fine.